lib/livebook/hubs/enterprise_client.ex

```diff
@@ -22,26 +25,35 @@ defmodule Livebook.Hubs.EnterpriseClient do
   @doc """
   Stops the WebSocket server.
   """
-  @spec stop(pid()) :: :ok
-  def stop(pid) do
-    pid |> GenServer.call(:get_server) |> GenServer.stop()
-    GenServer.stop(pid)
+  @spec stop(String.t()) :: :ok
+  def stop(id) do
+    if pid = GenServer.whereis(registry_name(id)) do
+      DynamicSupervisor.terminate_child(@supervisor, pid)
+    end
+
+    :ok
+  catch
+    :exit, _ -> :ok
```

Comment:
what throws this exit?

Reply:
It doesn't anymore, so a `rescue` would work for the `DynamicSupervisor.terminate_child/2`

Reply:
But I think it doesnt raise but return error tuple if the process does not exist.